Automattic/woocommerce-payments
# Handle minimum amount error in InitialState (project RPP).

Fixes #7415

#### Changes proposed in this Pull Request

- Create a new service `MinimumAmountService` to handle set and get transient, and get error message.
- Integrate this service into `InitialState`. 










#### Testing instructions





* Store with free shipping and flat rate (with something like over 1$)
* Create a product with 10 cents USD.
* Add this product to the checkout with free shipping.
* Try to make payment, see error message regarding the small amount. 
* Choose flat rate, which allows the order larger than minimum amount, and try to make payment again.
* No longer see the error.

Other tests:

* Add breakpoints to the new code in InitialState, and ensure they're run through. 

-------------------


**Post merge**
